src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/LogsPage/Blocks/actions.html.twig

```diff
@@ -0,0 +1,67 @@
+{#**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% trans_default_domain "Admin.Advparameters.Feature" %}
+<div class="action-buttons float-right">
+    {{ renderhook('displayDashboardToolbarIcons', {}) }}
+
+    <form action="{{ path('admin_logs_delete') }}" id="logs_delete_form" method="POST" class="form-inline float-left">
+        <input name="_method" type="hidden" value="DELETE" />
+        <input name="_token" type="hidden" value="{{ csrf_token('logs') }}" />
+        {% set message = 'Are you sure?'|trans({}, 'Admin.Notifications.Warning') %}
+        <a onclick="LogsPage.delete('{{ message }}');" href="javascript:void(0);">
+            {{ ps.tooltip(("Erase all"|trans), 'delete_forever') }}
+        </a>
+    </form>
+    <a id="desc-product-import" class="list-toolbar-btn" href="javascript:location.reload();">
+        {{ ps.tooltip(("Refresh list"|trans), 'refresh') }}
```

Comment:
Can I add domains here? It's classified in the messages folder.

Reply:
I can't use "messages" domain (by convention), so we need to move this translation into "Admin.Advparameters.Feature"

Reply:
Right, the question was can we add domains so that it does not fall into the messages folder.